www/front_src/src/PollerWizard/pollerStep1/index.tsx

```diff
@@ -39,33 +76,196 @@ const FormPollerStepOne = ({
       });
   };
 
+  const initializeFromRest = (value): void => {
+    setInitialized(true);
+    setInputTypeManual(!value);
+  };
+
+  const handleChange = (event): void => {
+    const { value, name } = event.target;
+
+    setError({
+      ...error,
+      [name]: value.trim() === '' ? 'required' : '',
+    });
+
+    setStepOneFormData({
+      ...stepOneFormData,
+      [name]: value,
+    });
+  };
+
+  const handleBlur = (event): void => {
+    const { value, name } = event.target;
+
+    setError({
+      ...error,
+      [name]: value.trim() === '' ? 'required' : '',
+    });
+  };
+
+  const handleSubmit = (event): void => {
+    event.preventDefault();
+
+    setWizard(stepOneFormData);
+    goToNextStep();
+  };
+
+  const nextDisabled =
+    Object.values(stepOneFormData).some((x) => x === '') ||
+    Object.values(error).some((x) => x !== '');
+
   React.useEffect(() => {
     getWaitList();
   }, []);
+  React.useEffect(() => {
+    if (waitList) {
+      const platform = waitList.find(
+        (server) => server.ip === stepOneFormData.server_ip,
+      );
+      if (platform) {
+        setStepOneFormData({
+          ...stepOneFormData,
+          server_name: platform.server_name,
+        });
+      }
+    }
+  }, [stepOneFormData.server_ip]);
 
-  const handleSubmit = (data): void => {
-    setWizard(data);
-    goToNextStep();
-  };
+  React.useEffect(() => {
+    if (isNil(waitList) || initialized) {
+      return;
+    }
+    initializeFromRest(waitList.length > 0);
+  }, [waitList]);
 
   return (
-    <Form
-      goToPreviousStep={goToPreviousStep}
-      initialValues={{}}
-      waitList={waitList}
-      onSubmit={handleSubmit}
-    />
+    <div>
+      <div className={classes.formHeading}>
+        <Typography variant="h6">{t('Server Configuration')}</Typography>
+      </div>
+      <form autoComplete="off" onSubmit={handleSubmit}>
+        <div className={classes.wizardRadio}>
+          <FormControlLabel
+            checked={inputTypeManual}
+            control={<Radio color="primary" size="small" />}
+            label={`${t('Create new Poller')}`}
+            onClick={(): void => setInputTypeManual(true)}
+          />
+          <FormControlLabel
+            checked={!inputTypeManual}
+            control={<Radio color="primary" size="small" />}
+            label={`${t('Select a Poller')}`}
+            onClick={(): void => setInputTypeManual(false)}
+          />
+        </div>
+        {inputTypeManual ? (
+          <div className={classes.form}>
+            <TextField
+              fullWidth
+              required
+              error={
+                error.server_name.length > 0 ? error.server_name : undefined
+              }
```

Comment:
It's better to extract the JSX into the body of the component to improve readability